easybuild/easyconfigs/v/VERSE/VERSE-0.1.5-foss-2016b.eb

```diff
@@ -0,0 +1,29 @@
+# This file is an EasyBuild reciPY, as per https://github.com/easybuilders/easybuild
+# Adam Huffman
+# The Francis Crick Institute
+easyblock = 'MakeCp'
+
+name = 'VERSE'
+version = '0.1.5'
+
+homepage = 'https://github.com/qinzhu/VERSE'
+description = """A versatile and efficient RNA-Seq read counting tool"""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+# No proper release tarballs
+sources = ['master.zip']
```

Comment:
@verdurin please use the new renaming support we have, to rename this to `SOURCE_ZIP`? Cfr. http://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html#alternative-formats-for-sources